go.mod

```diff
@@ -29,7 +29,7 @@ require (
 	github.com/docker/distribution v2.8.2+incompatible
 	github.com/docker/go-units v0.5.0
 	github.com/emicklei/go-restful/v3 v3.9.0
-	github.com/evanphx/json-patch v4.12.0+incompatible
+	github.com/evanphx/json-patch v5.6.0+incompatible
```

Comment:
`v5.6.0+incompatible` is *older* than `v4.12.0+incompatible` ... this dropped the following fix:

https://github.com/evanphx/json-patch/compare/v5.6.0...v4.12.0

do we care about this?